### DISCUSSION TITLE
J;ab go and Microsfot wireless bluetooth keyboards are not discoverable by linux mint ldme 6 cinnamon

### DISCUSSION THREAD

**Discussion by kadaken** [blueman-project/blueman#2563] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

I have tried connecting a  bluetooth microsoft keyboard and Jlab go keyboard, neither will be discovered by blueman. With the jlab go, there is an option for a dongle. it won't connect to that either, thought originally it says it was connecting and disconnecting with no indications on the keyboard.

I don't know how to pull reports and im not very savy with the terminal yet but am willing to do anything to get a keyboard working.
I am brand new to linux. running on a mid 2011 iMAC intel .

**Comment by cschramm** [comment]
The description of the JLab GO sounds like it may require Bluetooth LE and it is unlikely that a 2011 iMAC's builtin adapter supports that.
